packages/peregrine/package.json

```diff
@@ -38,14 +38,19 @@
     "@storybook/react": "~4.1.13",
     "concurrently": "~4.1.0",
     "intl": "~1.2.5",
+    "lodash.debounce": "~4.0.8",
```

Comment:
Is there a reason this is a `dependency` and a `peerDependency` ?

Reply:
Not a dependency at all anymore. 🙂